test/index.html

```diff
@@ -2,10 +2,10 @@
 <html>
   <head>
     <title>Vega Theme Test</title>
-    <script src="../node_modules/vega-lib/build/vega.js"></script>
```

Comment:
Why test and not dist or something?

Reply:
I should change that. should I change it to 

`Vega Theme Collection`.

what do you think?

Reply:
Nm, this is good.